docs/modules/ROOT/pages/how-to/zone-cluster-install.adoc

```diff
@@ -60,7 +60,21 @@ yq -i eval -i ".metadata.name = \"${APPUIO_CLOUD_ZONE}\"" \
 [source,bash,subs="attributes+"]
 ----
 yq -i eval '.classes += ["global.apps.appuio-cloud-zone"]'
-yq -i eval ".parameters.openshift4_terraform.terraform_variables.cluster_name = \"${APPUIO_CLOUD_ZONE}\" \
+yq -i eval ".parameters.openshift4_terraform.terraform_variables.cluster_name = \"${APPUIO_CLOUD_ZONE}\"" \
+  "${CLUSTER_ID}.yml"
+----
++
+.For cloudscale.ch
+[source,bash]
+----
+yq -i eval ".parameters.openshift4_terraform.terraform_variables.infra_flavor = \"plus-24\"" \
+  "${CLUSTER_ID}.yml"
+----
++
+.For Exoscale
```

Comment:
Please note that for Exoscale we never actually tested OCP4 cluster logging. Also, if we want to use Ceph RBD storage we need to size the storage cluster accordingly (see https://products.docs.vshn.ch/products/appuio/managed/ocp4_exoscale.html#_default_configuration_minimum_requirements "Storage Nodes").

Reply:
I took the `Huge` from the minimum requirements...

We should update the final requirements as soon as we install a productive Exoscale zone.